packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -88,15 +86,14 @@ and want to associate all stacks in the `App` scope to your imported application
 ```ts
 const app = new App();
 const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplication', {
-    applicationArnValue: 'arn:aws:servicecatalog:us-east-1:123456789012:/applications/applicationId',
-    stackProps: {
-        stackName: 'MyAssociatedApplicationStack',
-    },
+  applications: [appreg.ApplicationBuilder.importApplicationFromArn('arn:aws:servicecatalog:us-east-1:123456789012:/applications/applicationId', {
```

Comment:
We don't use *import* terminology for this anymore. "CDK Import" is overloaded enough as it is.

```suggestion
  applications: [appreg.ApplicationBuilder.existingApplicationFromArn('arn:aws:servicecatalog:us-east-1:123456789012:/applications/applicationId', {
```